content/api/hub/sample_release_bot_golang.md

```diff
@@ -0,0 +1,21 @@
+title = "Spin Plugin Release Bot - powered by Spin"
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "github", "actions"]
+
+[extra]
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Go"
+created_at = "2024-01-15T00:22:56Z"
+last_updated = "2024-01-15T00:22:56Z"
+spin_version = ">v1.5"
+summary =  "A bot automating releases for Spin Plugins - Powered by Spin"
+url = "https://github.com/rajatjindal/spin-plugin-release-bot"
+keywords = "github, golang, webhook"
```

Comment:
Should these keywords be aligned to the tags?  I am not sure how they are used.

Reply:
Need not be. 
https://github.com/fermyon/developer/pull/1141#discussion_r1456473759